chimney/src/main/scala/io/scalaland/chimney/internal/Constants.scala

```diff
@@ -0,0 +1,18 @@
+package io.scalaland.chimney.internal
+
+object Constants {
+  //proto enum
+  val ScalaPBEnumUnrecognizedInstanceName = "Unrecognized"
+  val ScalaPBGeneratedEnumBaseClassName = "scalapb.GeneratedEnum"
+
+  //proto oneof
+  val ScalaPBOneofEmptyInstanceName = "Empty"
+  val ScalaPBGeneratedOneofBaseClassName = "scalapb.GeneratedOneof"
+
+  //SDL @id
+  val SdlIdAnnotationFullName = "com.wixpress.infra.sdl.api.id.id"
+  val SdlIdGenerationTypeName = "IdGeneration"
+  val SdlIdGenerationManual = "Manual"
+  val SdlIdGenerationAuto = "Auto"
+  val SdlMissingIdPlaceholder = "ID_WILL_BE_GENERATED_BY_SDL"
```

Comment:
I'm not sure, how exactly this placeholder will impact the runtime. Just throwing ideas: what if there's some runtime validation at some point for a value to be a valid uuid? Maybe worth providing something like `00000000-0000-0000-0000-000000000000` instead?

Reply:
Good point. I believe we should promote good practices (i.e. using `java.util.UUID` in the domain object if one means to). That being said, I've introduced an additional check 
https://github.com/wix-incubator/chimney/pull/24/files#diff-22a5a9842c103e605ea378b01930735b290294f75a5dd0a29a348a04507d4a36R160
so this rule will apply only to `Option[String] -> String` cases